tensorflow_serving/g3doc/serving_advanced.md

```diff
@@ -143,7 +143,17 @@ With all these, `ServerCore` internally does the following:
   the `SessionBundleSourceAdapter`.
 
 Whenever a new version is available, this `AspiredVersionsManager` always
-unloads the old version and replaces it with the new one. If you want to
+unloads the old version and replaces it with the new one. The default behavior means that
+there will only be one version of the model served at a given time (and that version will be the 
+latest version). The latest version is determined from the model folder name which should be numeric.
+If you would like to serve all the versions available in `model_base_path`, you can set the following environment 
+variable before compiling Tensorflow Serving from source.
+
+~~~
+export TF_SERVING_MULTIPLE_VERSIONS=1
```

Comment:
i've removed the description on how to enable multiple versions.

The config protos has good comments on them. However it would be good if there's API docs or more details for advanced features. This would help users that intend to use tensorflow serving in production systems.

Perhaps along the line of dataflow docs or spark docs?
https://cloud.google.com/dataflow/docs/